src/test/java/api/support/http/BusinessLogicInterfaceUrls.java

```diff
@@ -10,10 +10,14 @@ public static URL items(String subPath) {
     return getUrl("/inventory/items", subPath);
   }
 
-  public static URL markWithdrawn(String subPath) {
+  public static URL markWithdrawnUrl(String subPath) {
```

Comment:
Why is the `Url` postfix useful? Is it to disambiguate it with other methods?

Reply:
Yes, the name conflicts with the fixture method